org.opent2t.sample.lamp.superpopular/com.insteon.lightbulb/js/README.md

```diff
@@ -0,0 +1,43 @@
+#Insteon Light Translator
+Translator forInsteon Light (http://www.insteon.com)
+
+## Setup Your Hardware
+Follow instructions in theInsteon app to set up your light withInsteon. This is a pre-requisite
```

Comment:
Fixed.

---
In reply to: [82906112](https://github.com/openT2T/translators/pull/107#discussion_r82906112) [](ancestors = 82906112)